config/config.go

```diff
@@ -134,8 +135,19 @@ type LDAPBind struct {
 	XXX map[string]interface{} `yaml:",inline"`
 }
 
+type LDAPQuery struct {
+	DN         string
+	Filter     string
+	Scope      string // Defaults to "(objectClass=*)".
```

Comment:
Wrong Comment, it's the filter not the scope

Reply:
indeed removed

Reply:
ldap lib have default value -> // Defaults to "(objectClass=*)".